client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -180,29 +183,20 @@ def get_curl(self):
             curl += f' -H "{header}: {value}"'
         return curl
 
-    # TODO: Removed deprecated `always_retry_404` and `always_retry_429` params
     def predict(
         self,
         x: List[Any],
         compress=False,
         max_retries: int = http_session.DEFAULT_MAX_RETRIES,
-        always_retry_404=False,
-        always_retry_429=False,
         retry_status: Set[int] = http_session.DEFAULT_STATUS_FORCELIST,
         backoff_factor: float = http_session.DEFAULT_BACKOFF_FACTOR,
         prediction_id: str = None,
     ) -> Dict[str, Any]:
         """
         Makes a prediction using input `x`.
 
-        .. deprecated:: 0.22.0
-           The ``always_retry_404`` parameter is deprecated. 404 is included by default in the
-           ``retry_status`` parameter.  Default is 13 retries over 10 minutes.  This behavior
-           can be changed by adjusting ``max_retries`` and ``backoff_factor``.
-        .. deprecated:: 0.22.0
-           The ``always_retry_429`` parameter is deprecated. 429 is included by default in the
-           ``retry_status`` parameter.  Default is 13 retries over 10 minutes.  This behavior
-           can be changed by adjusting ``max_retries`` and ``backoff_factor``.
+        .. versionchanged:: 0.23.0
+           The ``always_retry_404`` and ``always_retry_429`` parameters have been removed.
```

Comment:
We should include info about what they've been replaced by, like we did in the `deprecate` notes, for future readers (and/or people who missed the deprecation window)!

```rst
The ``always_retry_404`` and ``always_retry_429`` parameters have been removed. They are included by default in the ``retry_status`` parameter.  Default is 13 retries over 10 minutes.  This behavior can be changed by adjusting ``max_retries`` and ``backoff_factor``.
```

Reply:
Done